converter/src/main/java/gov/cms/qpp/conversion/model/validation/SupplementalData.java

```diff
@@ -0,0 +1,116 @@
+package gov.cms.qpp.conversion.model.validation;
+
+import gov.cms.qpp.conversion.model.TemplateId;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * An Enumeration of the known Supplemental Data values and types.
+ */
+public enum SupplementalData {
+
+	ALASKAN_NATIVE_AMERICAN_INDIAN("1002-5", SupplementalType.RACE),
+	ASIAN("2028-9", SupplementalType.RACE),
+	AFRICAN_AMERICAN("2054-5", SupplementalType.RACE),
+	HAWAIIAN_PACIFIC_ISLANDER("2076-8", SupplementalType.RACE),
+	WHITE("2106-3", SupplementalType.RACE),
+	OTHER_RACE("2131-1", SupplementalType.RACE),
+	HISPANIC_LATINO("2135-2", SupplementalType.ETHNICITY),
+	NOT_HISPANIC_LATINO("2186-5", SupplementalType.ETHNICITY),
+	MALE("M", SupplementalType.SEX),
+	FEMALE("F", SupplementalType.SEX),
+	MEDICARE("A", SupplementalType.PAYER),
+	MEDICAID("B", SupplementalType.PAYER),
+	PRIVATE_HEALTH_INSURANCE("C", SupplementalType.PAYER),
+	OTHER_PAYER("D", SupplementalType.PAYER);
+
+	private final String code;
+	private final SupplementalType type;
+
+	protected static final Map<SupplementalType, TemplateId> SUPPLEMENTAL_TYPES = new HashMap<>();
```

Comment:
EnumMap

Reply:
I'll take a look at this. Haven't really used EnumMap's.